src/pngtocss.c

```diff
@@ -70,26 +73,16 @@ static void print_error(const char *fname, status err)
 	char *emsg = "";
 
 	switch(err) {
-		case E_NOT_PNG:
-			emsg = "File is not a png";
-			break;
-		case E_NO_MEM:
-			emsg = "Out of memory";
-			break;
-		case E_INTERNAL:
-			emsg = "Problem inside libpng";
-			break;
-		case E_NO_FILE:
-			emsg = "Could not open file";
-			break;
 		case E_NOT_SUPPORTED:
-			emsg = "Gradient type not supported";
+			emsg = "pngtocss: Gradient type not supported";
 			break;
 		case OK:
 			return;
+		default:
+			return;
 	}
 
-	fprintf(stderr, "Error with ``%s''; %s\n", fname, emsg);
+	fprintf(stderr, "pngtocss: Error with ``%s''; %s\n", fname, emsg);
```

Comment:
might be better to use `argv[0]` just in case someone uses a different binary name or uses a symlink/hardlink